### PR TITLE
feat(progresscircle): gh-progresscircle

### PR DESCRIPTION
**Pull request by thomaspink**

### CODE REVIEW

File: src/demo-app/progress-circle/progress-circle-demo.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.progress-value {
```
Comment (ffriedl89) [inline src/demo-app/progress-circle/progress-circle-demo.scss]: not used anymore?

File: src/lib/progress-circle/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Progress circle
+
+`<gh-progress-circle>` are a circular indicators of progress and activity.
```
Comment (ffriedl89) [inline src/lib/progress-circle/README.md]: is a circular indicator for progress or activity

File: src/lib/progress-circle/progress-circle.spec.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+  it('should set the min, max and value aria attribute accordingly', () => {
+    const fixture = TestBed.createComponent(ValueProgressCircle);
+    fixture.detectChanges();
+
+    const progressElement = fixture.debugElement.query(By.css('gh-progress-circle'));
+    expect(progressElement.nativeElement.getAttribute('aria-valuemin')).toBe('100');
+    expect(progressElement.nativeElement.getAttribute('aria-valuemax')).toBe('200');
+    expect(progressElement.nativeElement.getAttribute('aria-valuenow')).toBe('150');
+  });
```
Comment (ffriedl89) [inline src/lib/progress-circle/progress-circle.spec.ts]: what about setting min > max ?

Comment (thomaspink) [reply src/lib/progress-circle/progress-circle.spec.ts]: Not really testable

**Pull request merged**